modules/distribution/pom.xml

```diff
@@ -153,6 +153,9 @@
 
                                 <replace file="target/wso2carbon-core-${carbon.kernel.version}/repository/conf/carbon.xml" token="&lt;/Security&gt;" value="${hideMenu}" />
 
+                                <!-- Adding WSO2 IS specific EnableAskPasswordAdminUI config to identity.xml -->
+                                <replace file="../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/identity.xml" token="&lt;/SelfRegistration--&gt;" value="&lt;/SelfRegistration--&gt;${line.separator}&lt;EnableAskPasswordAdminUI&gt;true&lt;/EnableAskPasswordAdminUI&gt;${line.separator}" />
+
```

Comment:
Better if we can have this property as false by default and making true here (rather than adding it new)